e2e/02-login-flow.e2e.spec.ts

```diff
@@ -29,11 +28,12 @@ describe("Login Flow", () => {
 
     const buildButton = await $(selector("Version Build Text", "StaticText"))
     await buildButton.waitForDisplayed({ timeout })
-    await buildButton.click()
-    await browser.pause(50)
-    await buildButton.click()
-    await browser.pause(50)
-    await buildButton.click()
+    let i = 0
+    while (i < 3) {
+      await buildButton.click()
+      await browser.pause(100)
+      i = i + 1
+    }
```

Comment:
the code is even longer with the loop. let's keep the loop unrolled and just replace the `50` to `100` is this is what fix this test